app/interactors/reservations/destroy.rb

```diff
@@ -0,0 +1,36 @@
+class Reservations::Destroy
+  include Interactor
+
+  before do
+    context.reservation_to_destroy ||= context.reservation
+  end
+
+  def call
+    destroy_reservation!
+  rescue StandardError => e
+    handle_errors(e.message)
+  end
```

Comment:
Nice destroy implementation. I was thinking though that we should also update the equipment reservations array to delete this specific date. That way, we allow users to reserve spots that are open when a previous user cancels the reservation

Reply:
Nice thinking!